### ISSUE TITLE
Improve importing

### ISSUE THREAD

**Issue by drphilmarshall** [DarkEnergyScienceCollaboration/SLTimer#31]
It seems crazy that we have to `import pycs` in all of our methods and functions. I am also uncomfortable in general with commands like `from X import *`, because it makes it harder to see where things come from. What's the advice from the DESC / LSST python style guide, @jchiang87 ?

**Comment by jchiang87** [comment]
The PEP 8 conventions say that imports should be at the top of the file.  If you are worried about clients polluting their namespace with `from desc.sltimer import *`, you can (and generally should) limit what's imported from your modules be setting the `__all__` variable:

```
import pycs
__all__ = ['read_in_db_data', 'read_in_tdc2_data']
```

In this case, `from desc.sltimer.IO import *` will only import the symbols listed in `__all__`.

**Comment by drphilmarshall** [comment]
That's interesting, I didn't know about `__all__`. Our problem seems to be that when we import individual classes or functions, or even `*`, from `desc.sltimer`, we get pycs import errors unless we put the import statements in each function. Wouldn't doing a global `import desc.sltimer` remove this need, and allow us to  put all import statements at the tops of the files?

**Comment by drphilmarshall** [comment]
Closed via #32!
